tests/wpt/mozilla/meta/css/get-computed-style-for-url.html.ini

```diff
@@ -0,0 +1,9 @@
+[get-computed-style-for-url.html]
+  type: testharness
+
+  [getComputedStyle(elem) for url() listStyle uses the resolved URL and elem.style uses the original URL]
+    expected: FAIL
```

Comment:
Let's add a `bug: https://github.com/servo/servo/issues/18015` line here and below.